src/patternfly/components/ModalBox/modal-box.scss

```diff
@@ -168,17 +118,10 @@
 
   // Base margin left and right equal for buttons when wrapped
   > .pf-c-button:not(:last-child) {
-    margin-right: calc(var(--pf-c-modal-box__footer--ButtonSpacer) / 2);
+    margin-right: var(--pf-c-modal-box__footer--c-button--sm--MarginRight);
```

Comment:
I think you got the vars reversed here? this one has `--sm` in it, and that should be in the `sm` breakpoint MQ.